cmd/ocp.go

```diff
@@ -93,6 +93,17 @@ func openShiftCmd() *cobra.Command {
 		} else {
 			envVars["ALERTS"] = ""
 		}
+		esServerEnv := os.Getenv("ES_SERVER")
+		esIndexEnv := os.Getenv("ES_INDEX")
+		if workloadConfig.MetricsEndpoint == "" && (esServerEnv == "" || esIndexEnv == "") {
+			envVars["ES_SERVER"] = ""
+			envVars["ES_INDEX"] = ""
+		}
+		if esServer != "" && esIndex != "" {
+			envVars["ES_SERVER"] = esServer
+			envVars["ES_INDEX"] = esIndex
+		}
```

Comment:
We shouldn't use external env vars

```suggestion
		if workloadConfig.MetricsEndpoint ==  "" && (esServer != "" && esIndex != "") {
			envVars["ES_SERVER"] = esServer
			envVars["ES_INDEX"] = esIndex
		} else{
			envVars["ES_SERVER"] = ""
			envVars["ES_INDEX"] = ""
		}
		
```

Reply:
This won't work when `MetricsEndpoint` is specified and has a `${ES_SERVER}` variable which is exported as an environment variable (.i.e in case of our CI tests), I beleive.

Reply:
I'll take a look at this later this week, yeh' I didn't think of the metrics-endpoint either and didn't factor that into my approach.

Reply:
`envVars["ES_SERVER"] = esServer` was all that was needed I thought, if you look at my initial patch. Let's hold this and I'll revisit this later this week. This is not a priority for me right now.

Reply:
Vishnu is right, we need to account for cases where metrics endpoitn is used and consumes variables from the env.

Reply:
Exporting the ES_SERVER and ES_INDEX env variables is a CPT/e2-benchmarking thing, these variables are not documented in kube-burner-ocp and their values are actually passed as flags `--es-server` and `--es-index` respectively

Reply:
We shouldn't read those variables from kube-burner-ocp

Reply:
This is not correct, we should use the flags

```
$ ES_SERVER="blabla" ES_INDEX=foo ./bin/amd64/kube-burner-ocp cluster-density-v2 --iterations=1  
time="2025-04-10 11:22:07" level=info msg="❤️  Checking for Cluster Health" file="cluster_health.go:46"
time="2025-04-10 11:22:09" level=info msg="Cluster is Healthy" file="cluster_health.go:54"
time="2025-04-10 11:22:11" level=info msg="📁 Creating opensearch indexer: indexer-0" file="metrics.go:67"
time="2025-04-10 11:22:11" level=fatal msg="Error creating indexer 0: OpenSearch health check failed: unsupported protocol scheme \"\"" file="metrics.go:70"
```